assessment/collectors/project_analyzer/file_dependency_analyzer.go

```diff
@@ -41,8 +42,23 @@ type GoDependencyAnalyzer struct {
 	BaseAnalyzer
 }
 
+func validateGoroot() error {
+
+	goroot := os.Getenv("GOROOT")
```

Comment:
please use loggers

Reply:
Print statement is not used. fmt is used to create Error object.